tikv/deploy-tikv-using-ansible.md

```diff
@@ -41,52 +34,289 @@ This guide describes how to install and deploy TiKV using Ansible. Ansible is an
 
     - Download the 2.0 GA version:
 
         ```bash
-        git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
+        $ git clone -b release-2.0 https://github.com/pingcap/tidb-ansible.git
         ```
     
     - Download the master version:
 
         ```bash
-        git clone https://github.com/pingcap/tidb-ansible.git
+        $ git clone https://github.com/pingcap/tidb-ansible.git
         ```
 
+    > **Note:** It is required to download `tidb-ansible` to the `/home/tidb` directory using the `tidb` user account. If you download it to the `/root` directory, a privilege issue occurs.
+
     If you have questions regarding which version to use, email to info@pingcap.com for more information or [file an issue](https://github.com/pingcap/tidb-ansible/issues/new).
 
-## Step 2: Install Ansible and the dependencies on the Control Machine
+## Step 4: Install Ansible and its dependencies on the Control Machine
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+It is required to use `pip` to install Ansible and its dependencies, otherwise a compatibility issue occurs. Currently, the TiDB 2.0 GA version and the master version are compatible with Ansible 2.4 and Ansible 2.5.
 
 1. Install Ansible and the dependencies on the Control Machine:
 
     ```bash
-    sudo yum -y install epel-release
-    sudo yum -y install python-pip curl
-    cd tidb-ansible
-    sudo pip install -r ./requirements.txt
+    $ cd /home/tidb/tidb-ansible
+    $ sudo pip install -r ./requirements.txt
     ```
 
-    Ansible and related dependencies are in the `tidb-ansible/requirements.txt` file.
+    Ansible and the related dependencies are in the `tidb-ansible/requirements.txt` file.
 
 2. View the version of Ansible:
 
     ```bash
-    ansible --version
+    $ ansible --version
+    ansible 2.5.0
+    ```
+
+## Step 5: Configure the SSH mutual trust and sudo rules on the Control Machine
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+1. Add the IPs of your target machines to the `[servers]` section of the `hosts.ini` file.
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ vi hosts.ini
+    [servers]
+    172.16.10.1
+    172.16.10.2
+    172.16.10.3
+    172.16.10.4
+    172.16.10.5
+    172.16.10.6
+
+    [all:vars]
+    username = tidb
+    ntp_server = pool.ntp.org
     ```
 
-    Currently, the 1.0 GA version depends on Ansible 2.4, while the 2.0 GA version and the master version are compatible with Ansible 2.4 and Ansible 2.5.
+2. Run the following command and input the `root` user account password of your target machines.
+
+    ```bash
+    $ ansible-playbook -i hosts.ini create_users.yml -u root -k
+    ```
+
+    This step creates the `tidb` user account on the target machines, configures the sudo rules and the SSH mutual trust between the Control Machine and the target machines.
```

Comment:
This step creates the `tidb` user account on the target machines **and** configures the sudo rules and the SSH mutual trust between the Control Machine and the target machines.